src/swagger-specs/query-service.yaml

```diff
@@ -972,14 +1286,14 @@ parameters:
     type: "string"
     in: "header"
     required: false
-    description: "A unique ID generated by Adobe.io"
+    description: "An optional header that serves as a tracking ID for the request. If one is not provided on the request, the server generates one automatically."
 
   User-Agent:
     name: "User-Agent"
     type: "string"
     in: "header"
     required: false
-    description: "type of the client making HTTP API request"
+    description: "The type of the client making the HTTP API request."
```

Comment:
I think it should be either just "HTTP request" or "API request" rather than a combination of the two. Prefer the former since it's a bit more general.